pallets/dkg-proposals/src/lib.rs

```diff
@@ -299,8 +275,12 @@ pub mod pallet {
 		ChainWhitelisted { chain_id: TypedChainId },
 		/// Proposer added to set
 		ProposerAdded { proposer_id: T::AccountId },
+		/// Queued proposer added to set
+		QueuedProposerAdded { proposer_id: T::AccountId },
 		/// Proposer removed from set
 		ProposerRemoved { proposer_id: T::AccountId },
+		/// Queued proposer removed from set
+		QueuedProposerRemoved { proposer_id: T::AccountId },
```

Comment:
Could you explain what `Queued` means here?

Reply:
Sorry this can be removed, artifact from previous attempt. Thank you for spotting.